packages/3d-web-avatar/src/helpers/findAssetsInCollection.ts

```diff
@@ -0,0 +1,43 @@
+import { CollectionDataType } from "@mml-io/3d-web-avatar-editor-ui";
```

Comment:
This function/file (`findAssetsInCollection`) should probably be in the `@mml-io/3d-web-avatar-editor-ui` package.

It shouldn't be in this package (`@mml-io/3d-web-avatar`) because this package doesn't have a dependency on the `@mml-io/3d-web-avatar-editor-ui` package.

Reply:
done